static/app/views/settings/projectPerformance/index.tsx

```diff
@@ -12,14 +10,12 @@ type Props = RouteComponentProps<{orgId: string; projectId: string}, {}> & {
   project: Project;
 };
 
-class ProjectPerformanceContainer extends Component<Props> {
-  render() {
-    return (
-      <Feature features={['performance-view']}>
-        <ProjectPerformance {...this.props} />
-      </Feature>
-    );
-  }
+function ProjectPerformanceContainer(props: Props) {
+  return (
+    <Feature features={['performance-view']}>
+      <ProjectPerformance {...props} />
+    </Feature>
+  );
 }
 
-export default withOrganization(ProjectPerformanceContainer);
+export default ProjectPerformanceContainer;
```

Comment:
The `ProjectSettingsLayout` already injects the organization object 

https://github.com/getsentry/sentry/blob/75163c99a58faaec2b011341550c99c137ece856/static/app/views/settings/project/projectSettingsLayout.tsx#L38